src/nms-api/nms-api.csproj

```diff
@@ -47,6 +47,24 @@ with the License.  You may obtain a copy of the License at
     <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
   </PropertyGroup>
 
+  <!-- SourceLink Start -->
+  <PropertyGroup>
+    <!-- Publish the repository URL in the built .nupkg (in the NuSpec <Repository> element) -->
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>

```

Comment:
Not sure about this line, as we already have link to the repository in the package via `RepositoryUrl` tag.

Reply:
According to the documentation [PublishRepositoryUrl](https://github.com/dotnet/sourcelink/blob/master/docs/README.md#publishrepositoryurl) modifies the same `RepositoryUrl` property. The only difference is that it does this dynamically.

I agree that there is no sense to have them both.
I'd rather remove `PublishRepositoryUrl` and do not touch the existing one (with explicit value). 

@Havret , please share your thoughts on this.

Reply:
Sounds good to me. 👍